packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -67,7 +67,9 @@ const VIcon = mixins(
   methods: {
     getIcon (): VuetifyIcon {
       let iconName = ''
-      if (this.$slots.default) iconName = this.$slots.default[0].text!.trim()
+      if (this.$slots.default && typeof this.$slots.default[0].text !=='undefined') {
```

Comment:
who about
```js
const iconname = this.$slots.default[0]?.text?.trim() ?? '';
```

Reply:
I believe the "?" operator works for null values only not for undefined ones,

https://replit.com/@LakhveerSingh/Testing-the-Non-Null-Operator

Reply:
?
![image](https://user-images.githubusercontent.com/15625235/122683387-14c12500-d1ff-11eb-821c-cf41390615b2.png)

Reply:
solved

Reply:
fixed linting error reported by CI for semicolon